src/components/pages/Volunteers/AddContactModal.tsx

```diff
@@ -0,0 +1,410 @@
+import React, { useState } from 'react';
+import { Button, Form, Modal, Col } from 'react-bootstrap';
+import { submitButton } from 'src/redux/modules/volunteerContacts';
+import { connect, ConnectedProps } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { RootState } from '../../../redux';
+
+interface AddContactModalProps {
+  volunteer: Volunteer;
+  show: boolean;
+  handleClose: () => void;
+  token: string;
+  orgId: number;
+}
+const mapStateToProps = (
+  state: RootState,
+  addContactModalProps: AddContactModalProps,
+) => ({
+  vcState: state.volunteerContacts,
+  volunteer: addContactModalProps.volunteer,
+  show: addContactModalProps.show,
+  handleClose: addContactModalProps.handleClose,
+  token: addContactModalProps.token,
+  orgId: addContactModalProps.orgId,
+  errors: state.volunteerContacts.errorMssg,
+});
+const mapDispatchToProps = (dispatch: Dispatch) =>
+  bindActionCreators(
+    {
+      submitButton,
+    },
+    dispatch,
+  );
+const connector = connect(mapStateToProps, mapDispatchToProps);
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+const AddContactModal: React.FC<PropsFromRedux> = ({
+  vcState,
+  volunteer,
+  show,
+  handleClose,
+  token,
+  orgId,
+  errors,
+  submitButton,
+}) => {
+  const STATES = [
```

Comment:
let's move this to the utils file perhaps?

Reply:
Moved!